AttitudeManager/Src/AM.cpp

```diff
@@ -1,3 +1,26 @@
 //
 // Created by Anthony Luo on 2022-10-12.
 //
+#include "AM.hpp"
+
+namespace AM {
+
+void AttitudeManager::runControlLoopIteration(
+    AttitudeManagerInput instructions) {
+    // Process Instructions
+
+    // Run Control Algorithms
+    for (auto controller : controller_interfaces) {
+        std::vector<ActuatorOutput> output =
+            controller->runControlsAlgo(instructions);
```

Comment:
```suggestion
            controller->runControlsAlgorithm(instructions);
```

Reply:
I like the full name, it's not that much longer and a lot more readable.